src/main/java/WorkShop01.kt

```diff
@@ -41,21 +56,31 @@ object WorkShop01 {
     // PS2. Оператор, который определяет остаток от деления - %
     // /
     // Для проверки задания необходимо запустить тесты в классе WorkShop01P5Test
-    fun evenOddNumber(number: Int) {
-        TODO()
+    fun evenOddNumber(number: Int) : String {
```

Comment:
Так как if умеет возвращать значение, то можем писать еще более лаконично 
```suggestion
    fun evenOddNumber(number: Int) : String = if (number % 2 == 0) {
```